### ISSUE TITLE
examples/typescript-vanilla doesn't work out-of-the box

### ISSUE THREAD

**Issue by atesgoral** [fastify/fastify-vite#181]
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the bug has not already been reported

### Fastify version

5.0.0

### Plugin version

6.0.7

### Node.js version

20.18.0

### Operating system

Windows

### Operating system version (i.e. 20.04, 11.3, 10)

Windows 10 + Ubuntu

### Description

```
npm run dev

✘ [ERROR] Missing "./plugin" specifier in "@fastify/vite" package [plugin externalize-deps]

    node_modules/esbuild/lib/main.js:1225:27:
      1225 │         let result = await callback({

  This error came from the "onResolve" callback registered here:

    node_modules/esbuild/lib/main.js:1150:20:
      1150 │       let promise = setup({

  The plugin "externalize-deps" was triggered by this import

    vite.config.js:2:28:
      2 │ import { viteFastify } from '@fastify/vite/plugin'
```

**Comment by onlywei** [comment]
This is fixed in 8.0.0-alpha.2

Please try that release.

**Comment by atesgoral** [comment]
Confirmed that it's working for me with that version. Thanks!
